### DISCUSSION TITLE
how to give ratings to recipes

### DISCUSSION THREAD

**Discussion by Chrysanthemum536** [TandoorRecipes/recipes#3184] [Q&A] [answered]
I don't see it anywhere in the dropdown or edit page
![image](https://github.com/TandoorRecipes/recipes/assets/104331720/a76ce798-ce73-4402-a8a9-57bb8031899d)

**Comment by smilerz** [comment, marked as answer]
Log cooking
![image](https://github.com/TandoorRecipes/recipes/assets/1808639/f39fec5d-8c78-4ddd-8ae8-df2204bc1460)
